lib/pbench/cli/agent/commands/triggers/list.py

```diff
@@ -0,0 +1,77 @@
+"""
+pbench-list-triggers
+
+This script will list all tool triggers from all tool groups, or list tool
+triggers from a specific group.
+
+"""
+import sys
+
+import click
+
+from pbench.cli.agent import CliContext, pass_cli_context
+from pbench.cli.agent.commands.triggers.base import TriggerCommand
+from pbench.cli.agent.options import common_options
+
+
+class TriggerList(TriggerCommand):
+    """List register triggers"""
+
+    def __init__(self, context):
+        super().__init__(context)
+
+    def execute(self):
+        if not self.pbench_run.exists():
+            # Silently exit if we don't have a pbench run directory.
+            return 1
+
+        if not self.context.group:
+            # list tool triggers in all groups
+            if len(self.groups) == 0:
+                self.logger.error("%s: error fetching list of tool groups", self.name)
```

Comment:
Can we add a set of unit tests for this which will test the `--help` and executing it once?  I am concerned that the logging needs to use `{}` style formatting instead of `%s`.